core-bundle/src/Framework/ContaoFramework.php

```diff
@@ -358,7 +361,7 @@ private function validateInstallation(): void
         }
 
         if (!$this->getAdapter(Config::class)->isComplete()) {
-            throw new RedirectResponseException('/contao/install');
+            throw new RedirectResponseException($this->urlGenerator->generate('contao_install', [], UrlGeneratorInterface::ABSOLUTE_URL));
```

Comment:
Why not inject the route prefix like in the `RobotsTxtListener` and redirect to `$this->routePrefix.'/install'`?

Reply:
We did not use the router before, either.